docs/tutorial/category_theory/calculating_colimits_in_Top.lean

```diff
@@ -24,19 +24,19 @@ def I_1 : pt ⟶ I :=
 { val := λ _, ⟨(1 : ℝ), begin rw [set.right_mem_Icc], norm_num, end⟩,
   property := continuous_const }
 
-def cylinder (X : Top) : Top := limit (pair X I)
+def cylinder (X : Top) : Top := prod X I
```

Comment:
Oof... I don't think there's any prospect of using `×`. Misusing `has_mul` here is awfully tempting.

If I were going to be responsible and just pick a new unicode character, perhaps it could be `⨱`. (At least in the fusion categories world, using an underline as "this is one category level higher than the usual notation" is becoming common.)

Reply:
Also, extremely evil, but there is a unicode symbol `⨯` (currently no shortcut) which is apparently different from `×` (`\times`).

Reply:
There's `∏` (\prod) and `∐` (\coprod) we could use for the indexed products and coproducts.

Reply:
They seem to work nicely:
```
example : Top := ∏ (λ n : ℕ, Top.of ℝ)
```

Reply:
@jcommelin, how do you feel about `⨱` for binary product? There's no analogous unicode symbol similar to `\oplus` `⊕`.

Perhaps just `×'` and `⊕'`?

Reply:
I like the second choice

Reply:
Oh... Actually, I don't know how to define notations that involve a single quote, because single quotes are parsed as delimiters when setting up notations!

Perhaps `×_` and `⊕_` are possibilities if we want to avoid exotic unicode?

Reply:
Done!